compiler/compile/OMRSymbolReferenceTable.cpp

```diff
@@ -610,46 +610,6 @@ OMR::SymbolReferenceTable::findOrCreateArrayTranslateAndTestSymbol()
    return element(arrayTranslateAndTestSymbol);
    }
 
-TR::SymbolReference *
-OMR::SymbolReferenceTable::findOrCreatelong2StringSymbol()
-   {
-   if (!element(long2StringSymbol))
-      {
-      TR::MethodSymbol * sym = TR::MethodSymbol::create(trHeapMemory(),TR_Helper);
-      sym->setHelper();
-
-      element(long2StringSymbol) = new (trHeapMemory()) TR::SymbolReference(self(), long2StringSymbol, sym);
-      }
-   return element(long2StringSymbol);
-   }
-
-
-TR::SymbolReference *
-OMR::SymbolReferenceTable::findOrCreatebitOpMemSymbol()
-   {
-   if (!element(bitOpMemSymbol))
-      {
-      TR::MethodSymbol * sym = TR::MethodSymbol::create(trHeapMemory(),TR_Helper);
-      sym->setHelper();
-
-      element(bitOpMemSymbol) = new (trHeapMemory()) TR::SymbolReference(self(), bitOpMemSymbol, sym);
-      }
-   return element(bitOpMemSymbol);
-   }
-
-TR::SymbolReference *
-OMR::SymbolReferenceTable::findOrCreateSinglePrecisionSQRTSymbol()
```

Comment:
This symbol reference is coupled with code gen support for single precision square root (see `supportsSinglePrecisionSQRT` in the OMRCodeGenerator.cpp files. If the codegen implementation is going to remain in OMR this symbol reference creation function should remain since it is required to use the codegen implementation in OMR.